### PR TITLE
Fix offline build for test binaries

### PR DESCRIPTION
**Pull request by bobtwinkles**

### CODE REVIEW

**Comment by bobtwinkles** [comment]
I'm not sure `task-test` is actually in the right place because it probably won't work in most dbt environments (it needs a host compiler, which we don't ship). Maybe it should go in `contrib` or something instead.

**Comment by m-m-adams** [comment]
Seems fine to me, we don't need any special compiler or libraries and I think having a host compiler to build the test binary is a pretty small requirement

File: scripts/dbt/project.py [hunk 1]
```diff
@@ -1,47 +0,0 @@
-import os
-from SCons.Script import GetOption
-from dbt.util import vcheck, vprint
-
-
-def compose_valid_targets(cmd_env=None):
```
Comment (m-m-adams) [inline scripts/dbt/project.py:6]: Is this file just unused? It doesn't seem related to tests

Comment (bobtwinkles) [reply scripts/dbt/project.py:6]: yeah, this is for the old SCons build. I just noticed it while adding the build task, it should probably technically be a separate PR...

**Review by m-m-adams — APPROVED** [review]
